packages/api/src/routes/medical/internal-cq.ts

```diff
@@ -0,0 +1,93 @@
+import { Carequality } from "@metriport/carequality-sdk/client/carequality";
+import { normalizeOid } from "@metriport/shared";
+import dayjs from "dayjs";
+import duration from "dayjs/plugin/duration";
+import { Request, Response } from "express";
+import Router from "express-promise-router";
+import httpStatus from "http-status";
+import {
+  CQDirectoryOrg,
+  createCQOrganization,
+} from "../../command/medical/cq-directory/create-cq-organization";
+import {
+  getCoordinates,
+  getState,
+  getUrls,
+} from "../../command/medical/cq-directory/parse-cq-organization";
+import { Config } from "../../shared/config";
+import { asyncHandler } from "../util";
+
+const apiKey = Config.getCQApiKey();
+const apiMode = Config.getEnvType();
+
+dayjs.extend(duration);
+
+const router = Router();
+
+router.post(
+  "/test",
+  asyncHandler(async (req: Request, res: Response) => {
+    const testOrg = {
+      oid: "1.1.1.11123",
+      urlXCPD: "https://api.app.com/1",
+      urlDQ: "https//api.app.com/2",
+      urlDR: "https//api.app.com/3",
+      name: "Test Org",
+      latitude: "40.689247",
+      longitude: "-74.044502",
+      state: "NY",
+    };
+
+    const resp = await createCQOrganization(testOrg);
+    return res.status(httpStatus.OK).json(resp);
+  })
+);
+
+router.post(
+  "/",
+  asyncHandler(async (req: Request, res: Response) => {
```

Comment:
what is this doing? let's describe this route, and the inputs/returns, with a function header